src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculate {
+    private double side;
```

Comment:
side it's not a radius, so it is not common state for every figure.  Lets left here only color, and remove side parameter